### DISCUSSION TITLE
Running noisetorch as a service on startup.

### DISCUSSION THREAD

**Discussion by yash10019coder** [noisetorch/NoiseTorch#356] [Q&A (How can I ...?)] [answered]
Can we like run noisetorch as a service on startup in linux rather than like opening it manually ourselves?

**Comment by yash10019coder** [comment, marked as answer]
Found in wiki thanks

**Reply by TheDukeofErl** [reply]
Just for posterity's sake, I'm going to drop a link to the systemd instructions [here](https://github.com/noisetorch/NoiseTorch/wiki/Start-automatically-with-Systemd).

There's also some discussion of an alternate way to do this, instead automatically opening the program after a delay on login here:  https://github.com/noisetorch/NoiseTorch/issues/342

**Reply by yash10019coder** [reply]
thanks @TheDukeofErl
